lib/realtime/vehicle.ex

```diff
@@ -150,8 +180,11 @@ defmodule Realtime.Vehicle do
   @spec is_a_timepoint?(StopTime.t()) :: boolean
   defp is_a_timepoint?(%StopTime{timepoint_id: timepoint_id}), do: timepoint_id != nil
 
-  @spec decode_current_status(String.t()) :: current_status()
+  @spec decode_current_status(String.t() | atom()) :: current_status()
   defp decode_current_status("IN_TRANSIT_TO"), do: :in_transit_to
+  defp decode_current_status(:IN_TRANSIT_TO), do: :in_transit_to
   defp decode_current_status("INCOMING_AT"), do: :in_transit_to
+  defp decode_current_status(:INCOMING_AT), do: :in_transit_to
   defp decode_current_status("STOPPED_AT"), do: :stopped_at
+  defp decode_current_status(:STOPPED_AT), do: :stopped_at
```

Comment:
Which data source is making strings and which data source is making atoms? Having both makes me think we might be doing two different parsing steps; maybe they could be consolidated?

Reply:
Concentrate's parser converts everything to an atom.

Reply:
Good catch, for a while I was supporting both the concentrate format and the old JSON parsing (which required the string format), but now the string format is vestigial.